packages/ui/src/components/va-sidebar/VaSidebarItem/VaSidebarItem.vue

```diff
@@ -58,11 +56,9 @@ export default defineComponent({
       }
 
       if (props.active) {
-        const border = props.borderColor === undefined ? props.activeColor : props.borderColor
-
-        style['border-color'] = getColor(border)
+        style['border-color'] = getColor(props.borderColor === undefined ? props.activeColor : props.borderColor)
```

Comment:
`props.borderColor === undefined` can be replaced with `!props.borderColor` if there is no sense in `undefined`.

Reply:
User can pass in `props.borderColor` an empty string and `getColor` can process an empty string. So, I think, we have to check if `props.borderColor` doesn't define with user.